### ISSUE TITLE
Autoimmune gastritis

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6074]
**Preferred term label:**
Autoimmune gastritis

**Synonyms**
AIG

**Definition (free text, please give PubMed ID)**
Autoimmune gastritis is a chronic inflammatory disease with destruction of parietal cells of the corpus and fundus of the stomach
PMID: 27671008

**Parent term (use hpo.jax.org/app)**
Gastritis, Autoimmunity

**Comment by pnrobinson** [comment]
This is a disease and not a phenotypic feature. We would use the MONDO term, https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0031014
